proposals/runtime_get_contexts.md

```diff
@@ -236,6 +238,22 @@ This is an artifact of existing APIs and precedence. Since many existing APIs
 use the constant integer values, we want to be consistent with those. However,
 for newly-introduced fields, we use the more intuitive `undefined` state.
 
+
+### Developer Tools Contexts
+
+Extensions can use the
+[Dev Tools API](https://developer.chrome.com/docs/extensions/mv3/devtools/) to
+extend the browser's developer tools. When doing so, these extensions can run
+different types of contexts (page, panel, sidebar). `runtime.getContexts()`
+returns these contexts with the `DEVELOPER_TOOLS` context type.
+
+- These contexts have a `tabId` of -1 as these contexts are not hosted inside
```

Comment:
Is `frameId` set? I don't have strong feelings here.

Reply:
+1 to the question about frame ID.

Also, would it make sense to have tab ID be non-null if the devtools are docked to the tab?

(Similarly, no very strong feelings)

Reply:
I too do not have very strong feelings here. I would want to know what other's think about this. Here are my thoughts:

1. From what I understand, `tabId` is set for contexts which are running inside a tab. Developer tools contexts do not run in a tab. They run in the developer tools window. 
2. Developer tools contexts can be background contexts too. Setting a `tabId` for such contexts sounds unnatural. 

Since `frameId` is used to identify a frame within a tab, we should set it to -1 too.

Reply:
Agree with @sohailrajdev97. In addition, getting the TabId is already possible using `devtools.inspectedWindow.tabId` https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/devtools/inspectedWindow/tabId from the devtools panel itself. Which can be requested using messaging between extension contexts.